Apps/WebClient/src/ClientApp/src/components/credential/credentialList.vue

```diff
@@ -0,0 +1,208 @@
+<script lang="ts">
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { faSyringe } from "@fortawesome/free-solid-svg-icons";
+import Vue from "vue";
+import { Component } from "vue-property-decorator";
+import { Getter } from "vuex-class";
+
+import MessageModalComponent from "@/components/modal/genericMessage.vue";
+import { CredentialState, WalletCredential } from "@/models/credential";
+import { DateWrapper } from "@/models/dateWrapper";
+import { ImmunizationEvent } from "@/models/immunizationModel";
+import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
+import container from "@/plugins/inversify.container";
+import { ILogger } from "@/services/interfaces";
+
+library.add(faSyringe);
+
+interface Entry {
```

Comment:
nit: Entry sounds too general for me. Maybe CredentialEntry?

Reply:
Sounds good, will update.